en/uavcan/README.md

```diff
@@ -1,70 +1,92 @@
-	:::
+* [ESC/Motor controllers](../uavcan/escs.html)
 * Airspeed sensors
   * [Thiemar airspeed sensor](https://github.com/thiemar/airspeed)
 * GNSS receivers for GPS and GLONASS
   * [Zubax GNSS](https://zubax.com/products/gnss_2)
 * Power monitors
-  * [Pomegranate Systems Power Module](../power_module/pomegranate_systems_pm.md)
-  * [CUAV CAN PMU Power Module](../power_module/cuav_can_pmu.md)
-
-In contrast to hobby-grade devices it uses rugged, differential signalling and supports firmware upgrades over the bus.
-All motor controllers provide status feedback and implement field-oriented-control \(FOC\).
+  * [Pomegranate Systems Power Module](../uavcan/pomegranate_systems_pm.md)
+  * [CUAV CAN PMU Power Module](../uavcan/cuav_can_pmu.md)
+* Distance sensors
+  - [Avionics Anonymous Laser Altimeter UAVCAN Interface](../uavcan/avanon_laser_interface.md)
 
 :::note
-PX4 requires an SD card for UAVCAN node allocation and firmware upgrade.
-It is not used during flight by UAVCAN.
+PX4 does not support UAVCAN servos (at time of writing).
 :::
 
-## Initial Setup
 
-The following instructions provide a step-by-step guide to connect and setup a quadcopter with ESCs and GPS connected via UAVCAN.
-The hardware of choice is a Pixhawk 2.1, Zubax Orel 20 ESCs and a Zubax GNSS GPS module.
+## Wiring
 
-### Wiring
+All UAVCAN components share the same connection architecture/are wired the same way.
+Connect all on-board UAVCAN devices into a chain and make sure the bus is terminated at the end nodes (the order in which the nodes are connected/chained does not matter).
 
-The first step is to connect all UAVCAN enabled devices with the flight controller.
-The following diagram displays how to wire all components.
-The used Zubax devices all support a redundant CAN interface in which the second bus is optional but increases the robustness of the connection.
+The following diagram shows this for a flight controller connected to [UAVCAN motor controllers (ESCs)](../uavcan/escs.html) and a UAVCAN GNSS.
 
 ![UAVCAN Wiring](../../assets/uavcan/uavcan_wiring.png)
 
-It is important to mention that some devices require an external power supply \(e.g. Zubax Orel 20\) and others can be powered by the CAN connection \(e.g Zubax GNSS\) itself.
-Please refer to the documentation of your hardware before continuing with the setup.
+The diagram does not show any power wiring.
+Refer to your manufacturer instructions to confirm whether components require separate power or can be powered from the CAN bus itself. 
+
+For more information about proper bus connections see [UAVCAN Device Interconnection](https://kb.zubax.com/display/MAINKB/UAVCAN+device+interconnection) (Zubax KB).
+
+:::note
+- While the connections are the same, the _connectors_ may differ across devices.
+- An second/redundant" CAN interface may be used, as shown above (CAN2).
+  This is optional, but can increase the robustness of the connection.
+:::
+
+
+## PX4 Configuration
```

Comment:
@bkueng This is main change since last time. It makes clear when you select each of the UAVCAN_ENABLE options. Because you will have to manually configure any nodes that don't support auto config, this implies that you only use manual config if NONE of the modes support auto.